src/util/config.js

```diff
@@ -1,15 +1,21 @@
 // @flow
 
+import browser from './browser';
+
 type Config = {|
   API_URL: string,
   REQUIRE_ACCESS_TOKEN: boolean,
-  ACCESS_TOKEN: ?string
+  ACCESS_TOKEN: ?string,
+  DEVICE_PIXEL_RATIO: number,
 |};
 
 const config: Config = {
     API_URL: 'https://api.mapbox.com',
     REQUIRE_ACCESS_TOKEN: true,
-    ACCESS_TOKEN: null
+    ACCESS_TOKEN: null,
+    get DEVICE_PIXEL_RATIO() {
```

Comment:
Why use a getter method instead of just assigning the property directly? Again, I'd err on the side of consistency and simplicity here unless there's a specific reason not to.

Reply:
Hi! The reason behind this was to save the behavior of browser.getDevicePixelRatio:
```js
get devicePixelRatio() { return window.devicePixelRatio; },
```
so that if pixel ratio wasn't set in config it would use current value of window.devicePixelRatio and not the value that was present at the moment when config object was created.

It's probably not a big deal. I can't think of a use case when pixel ratio would suddenly change (apart from some unit tests). So I am going to stick to the simpler solution.